### PR TITLE
Adding all files for wordFrequency

### PR DESCRIPTION
**Pull request by omarwaheed2**

### CODE REVIEW

File: word_frequency_java/WordFrequency.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                }
+            }
+        }
+    }
+
+    public void printTable(){
+        System.out.println("\t Word \t Frequency");
+        int totalWords = 0;
+        for ( String word : wordFrequency.keySet() ){
+            System.out.println("\t " + word + " \t " + wordFrequency.get(word));
```
Comment (rehanrasool) [inline word_frequency_java/WordFrequency.java]: General advice:
Use System.out.printf() instead to make the formatting easier to read e.g.
`System.out.printf("\t %s \t %d\n",  word,  wordFrequency.get(word));`
[have not tested]

File: word_frequency_java/WordFrequency.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        }
+        System.out.println("Total Words in the table : " + totalWords);
+    }
+
+
+    public static boolean isValidWord(String word){
+        for (char c: word.toCharArray()){
+            int ascii = c;
+            if ( (ascii < 97 || ascii > 122) &&
+                 (ascii < 65 || ascii > 90)
```
Comment (rehanrasool) [inline word_frequency_java/WordFrequency.java]: For better readability, most languages allow something like this:
`if ( (c < 'a' || ascii > 'Z' ) && ( c < 'A' || ascii > 'Z') `
[you might have to cast to int]

File: word_frequency_java/WordFrequency.java (hunk 1 above)
Comment (omarwaheed2) [reply word_frequency_java/WordFrequency.java]: Yeah makes sense, using printf

File: word_frequency_java/WordFrequency.java (hunk 2 above)
Comment (omarwaheed2) [reply word_frequency_java/WordFrequency.java]: Good call, changed

**Review by rehanrasool — APPROVED** [review]
Looks good!

**Pull request merged**